addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6987,6 +6987,10 @@ registry.BackgroundShape = SnippetOptionWidget.extend({
             this._lastShapePalette = palette;
             this._shapeBackgroundImagePerClass = {};
             for (const styleSheet of this.$target[0].ownerDocument.styleSheets) {
+                if (styleSheet.href && new URL(styleSheet.href).origin !== location.hostname)   {
```

Comment:
Isn't the rule also infringed if a different port number is accessed ?

Reply:
Good question, I just discover this constraint so I need to search

Reply:
Tested in local with a small express js server and a dummy css, hostname still bug
Using host works

domain = host = hostname + port